pkg/csi_driver/node.go

```diff
@@ -276,6 +282,26 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	return &csi.NodePublishVolumeResponse{}, nil
 }
 
+func removeDisallowedMountOptions(fuseMountOptions []string, disallowedFlags map[string]bool) []string {
```

Comment:
nit, I think this can go in pkg/csi_driver/utils.go as its a utility function

Please add function comment